Source/WebCore/rendering/RenderObject.cpp

```diff
@@ -2384,6 +2384,15 @@ void RenderObject::RepaintRects::transform(const TransformationMatrix& matrix, f
         *outlineBoundsRect = LayoutRect(encloseRectToDevicePixels(matrix.mapRect(*outlineBoundsRect), deviceScaleFactor));
 }
 
+PointerEvents RenderObject::usedPointerEvents() const
+{
+    // FIXME: This could be a performance problem, checking this all
```

Comment:
`renderingSuppressedForViewTransition()` is just checking a bit so it's probably fine?